charger/ocpp.go

```diff
@@ -168,6 +169,14 @@ func NewOCPP(id string, connector int, idtag string,
 	// see who's there
 	if boot {
 		conn.TriggerMessageRequest(core.BootNotificationFeatureName)
+		select {
+		case <-time.After(timeout):
+			c.log.DEBUG.Printf("BootNotification timeout")
+		case res := <-cp.BootNotificationRequest():
+			if res != nil {
```

Comment:
Das if bringt nix- c.bootNotofication ist in jedem Fall nil. Der Chek muss in Diagnosis- der Diagnosis Teil fehlt hier um das sinnvoll zu verwenden?

Reply:
Aua, der war gut...

Reply:
Jetzt aber.